jubilant/statustypes.py

```diff
@@ -254,6 +254,26 @@ def is_waiting(self) -> bool:
         """Report whether the application status for this app is "waiting"."""
         return self.app_status.current == 'waiting'
 
+    @property
+    def leader_unit(self) -> UnitStatus:
+        """Return the leader unit's status.
+
+        For example, instead of duplicating the application name::
```

Comment:
I feel like "unit 0 may not be leader" is a more important reason to use this property than avoiding a little bit of duplication. Could we adjust the help to mention it?